test/image/mocks/zz01-plot_types_grid_dash.json

```diff
@@ -0,0 +1,243 @@
+{
```

Comment:
Please rename this file to `z-plot_types_grid_dash.json`. There should be no need for `zz01`.